core/eolearn/tests/test_core_tasks.py

```diff
@@ -426,10 +426,10 @@ def test_extract_bands(test_eopatch):
     patch = move_bands(test_eopatch)
     assert np.array_equal(patch.data["MOVED_BANDS"], patch.data["REFERENCE_SCENES"][..., bands])
 
+    tmp = patch.data["MOVED_BANDS"][0, 0, 0, 0]
     patch.data["MOVED_BANDS"][0, 0, 0, 0] += 1.0
-    assert math.isclose(
-        patch.data["REFERENCE_SCENES"][0, 0, 0, bands[0]] + 1.0, patch.data["MOVED_BANDS"][0, 0, 0, 0], rel_tol=0.2
-    )
+    assert_equal(patch.data["REFERENCE_SCENES"][0, 0, 0, bands[0]], tmp)
```

Comment:
these should be identical values, so a simple `==` comparison should suffice, no?